src/services/types/Service.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Interface for sending data to different platforms
+ *
+ * @template Configuration - type of configuration
+ */
+export interface Service<Configuration> {
+  /**
+   * Client config
+   *
+   * @param {Configuration} configuration - Client configuration.
+   */
+  configuration?: Configuration;
```

Comment:
looks like configuration should be a private property

Reply:
When i do it private i get typescript error: 'private' modifier cannot appear on a type member 
https://github.com/microsoft/TypeScript/issues/25163

Reply:
that is right, just remove it. We don't need to know about it from outside